docs/LTS.md

```diff
@@ -25,6 +25,18 @@ out in this document:
 
 A "month" is defined as 30 consecutive days.
 
+> ## Security Releases and Semver
+>
+> As a consequence of providing long-term support for major releases, there
+> are occasions where we need to release a breaking changes as a _minor_
+> version release. Such changes will _always_ be noted in the
+> [release notes](https://github.com/fastify/fastify/releases).
+>
+> To avoid automatically updating to new patch releases we recommend using
+> a tilde (`~`) range qualifier. For example, to get patches for the 3.15
+> release, and avoid automatically updating to the 3.16 release, specify
+> the dependency as `"fastify": "~3.15.x"`.
```

Comment:
```suggestion
> To avoid automatically receiving breaking security updates we recommend
> using a tilde (`~`) range qualifier. For example, to get patches for the 3.15
> release, and avoid automatically updating to the 3.16 release, specify
> the dependency as `"fastify": "~3.15.x"`. We do not recommend this approach.
```

Reply:
> We do not recommend this approach

^ directly contradicts the first sentence in the paragraph:

> To avoid automatically receiving breaking security updates we recommend
> using a tilde (`~`) range qualifier.

Reply:
I updated the comment

Reply:
I tweaked it a bit and committed. Please re-review.